sdks/java/core/src/main/java/org/apache/beam/sdk/schemas/utils/ReflectUtils.java

```diff
@@ -82,6 +82,7 @@ public static List<Method> getMethods(Class<?> clazz) {
               .filter(m -> !Modifier.isPrivate(m.getModifiers()))
               .filter(m -> !Modifier.isProtected(m.getModifiers()))
               .filter(m -> !Modifier.isStatic(m.getModifiers()))
+              .sorted((m1, m2) -> m1.getName().compareTo(m2.getName()))
```

Comment:
Nit: would it be better use some sorted data structure like SortedSet or immutable sorted set

To prevent callers from mutating the sorting order